src/hotspot/share/runtime/abstract_vm_version.cpp

```diff
@@ -138,41 +138,42 @@ const char* Abstract_VM_Version::vm_vendor() {
 
 
 const char* Abstract_VM_Version::vm_info_string() {
```

Comment:
For future benefit, how about also adding a comment explain why we avoid dynamic memory allocation for the vm_info_string here?

Reply:
I have add a comment, but my English is not very well. If you have proper description, please let me known.

Reply:
Looks ok to me, thanks. Please update the contributor properly since the change is from https://github.com/openjdk/jdk/pull/24171/commits/baff6b166d130c9adeecfb9f2b418d86322d4826.

Reply:
Okey, that was my original plan also.

Reply:
Sounds good to me!